src/Auth0.php

```diff
@@ -37,7 +37,17 @@ public function getSdk(): \Auth0\SDK\Auth0
     }
 
     /**
-     * Create/return instance of the Auth0-PHP SdkConfiguration.
+     * @inheritdoc
```

Comment:
Added to ease unit testing purposes, allows overwriting the Auth0-PHP SDK instance.